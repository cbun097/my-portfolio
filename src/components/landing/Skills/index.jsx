import React from 'react'
import AnchorLink from 'react-anchor-link-smooth-scroll'
import { Container, Button } from 'components/common'
import dev from 'assets/illustrations/skills.svg'
import { Wrapper, SkillsWrapper, Details, Thumbnail } from './styles'

export const Skills = () => (
  <Wrapper id="about">
    <SkillsWrapper as={Container}>
      <Thumbnail>
        <img src={dev} alt="I’m John and I’m a Backend & Devops engineer!" />
      </Thumbnail>
      <Details>
        <h1>Hi There!</h1>
        <p>
          My name is Claire Bun. I am a DEC (Quebec's diploma of college studies) in Information Technology. Currently,
          I am an undergraduate in Computer Science. I have mostly experience in Angular and ReactJs.
        </p>
        <Button as={AnchorLink} href="#contact">
          Hire me
        </Button>
      </Details>
    </SkillsWrapper>
  </Wrapper>
)
