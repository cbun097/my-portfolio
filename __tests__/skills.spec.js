import React from 'react'
import renderer from 'react-test-renderer'
import { Skills } from '../src/components/landing/Skills/index'

describe('Project Component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Skills />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
