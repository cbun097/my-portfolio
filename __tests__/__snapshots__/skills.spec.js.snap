// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Component renders correctly 1`] = `
<div
  className="sc-fzqBZW hBpDZs"
  id="about"
>
  <div
    className="sc-fzpans cLzOFF sc-fzqNJr PzOZk"
  >
    <div
      className="sc-fzoLag kdSrNb"
    >
      <img
        alt="I’m Claire and I’m a Frontend developer and student!"
        src="test-file-stub"
      />
    </div>
    <div
      className="sc-fzoyAV dHUstq"
    >
      <h1>
        Hi There!
      </h1>
      <p>
        My name is Claire Bun. I am a DEC (Quebec's diploma of college studies) in Information Technology. Currently, I am an undergraduate in Computer Science. I have mostly experience in Angular and ReactJs.
      </p>
      <a
        className="sc-fzplWN jnWZni"
        href="#contact"
        onClick={[Function]}
      >
        Hire me
      </a>
    </div>
  </div>
</div>
`;
